authenticator.py

```diff
@@ -24,6 +26,10 @@ class Authenticator(object):
     def _configure_users(cls, users):
         cls._users = users
 
+    @classmethod
+    def _configure_unhashed(cls, unhashed):
+        cls._unhashed = unhashed
```

Comment:
Think we should name this variable `cls._allow_unhashed` to be more explicit, I wasn't sure what it meant at first.